### DISCUSSION TITLE
Retrofitting microsymfony in an existing app

### DISCUSSION THREAD

**Discussion by tacman** [strangebuzz/MicroSymfony#8] [Ideas]
I like the stack that microsymfony creates, and am wondering if there's a way to retrofit it into existing projects.

For example, if I run symfony new --demo, or symfony new --webapp, and I made a bunch of changes, and now I want to incorporate the microsymfony stack, is there a way to do that?

Unlike a bundle, which has recipes and commands, this is a template that is used to start a project, but in some ways it feels like a toolkit that could be used to bootstrap a project, including retrofitting an existing one.

In other words, can microsymfony be recreated as a list of commands that could be used in an existing project?

```bash
# these are psuedo commands

# install castor
curl microsymfony.repo/castor.php (get the default tasks)
curl microsymfony.repo/base.html.twig templates/base.html.twig (copy the default layout)
composer req symfony/asset-mapper
bin/console importmap:install basecss
```

Of course, some of these feel like tasks!  We could bootstrap castor and then provide tasks to add the rest of the framework.

This is, or course, simple brainstorming.  I have some demo applications that I'm converting to use AssetMapper (to get rid of the front end build step), and would like to simplify the layout and add tasks at the same time.

**Comment by COil** [comment]
Hello, I don't think a template aims to handle this. 
But if you started a project with MicroSymfony and want the latest commit, you still can pull commits from the main branch.
